Null value for OID

Sometimes, we have these error messages in the js console. Not sure what specific behaviour precedes or follows this.

```
linkeddata-service-won.js: fetching:        https://satsrv06.researchstudio.at/won/resource/need/4508223599103297500/connections/
app_jspm.bundle.js:60591 error in lexicon retrieving OID:
app_jspm.bundle.js:60592 403
app_jspm.bundle.js:60601 Null value for OID
undefined:1 Uncaught (in promise) Error: Failed to fetch connection uris for need <https://satsrv06.researchstudio.at/won/resource/need/4508223599103297500> because of:
Unknown retrieving OID in lexicon:403
    at Lexicon.Lexicon.retrieve (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:60603:15)
    at QueryEngine.QueryEngine.denormalizeBindings (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84604:36)
    at QueryEngine.QueryEngine.denormalizeBindingsList (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84561:27)
    at https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84696:33
    at QueryEngine.QueryEngine.executeSelect (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84797:15)
    at QueryEngine.QueryEngine.executeQuery (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84693:14)
    at QueryEngine.QueryEngine.execute (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84636:16)
    at Store.Store.graph (https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:86781:19)
    at https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:88692:55
    at https://satsrv06.researchstudio.at/owner/generated/app_jspm.bundle.js:84731:17
```

--- a/webofneeds/won-owner-webapp/src/main/webapp/app/service/linkeddata-service-won.js
+++ b/webofneeds/won-owner-webapp/src/main/webapp/app/service/linkeddata-service-won.js
@@ -1065,7 +1065,7 @@ const rdfstore = window.rdfstore;
      * @return {*} the data of all connection-nodes referenced by that need
      */
     won.getConnectionsOfNeed = (needUri, requesterWebId = needUri) =>
-        won.getConnectionUrisOfNeed(needUri)
+        won.getConnectionUrisOfNeed(needUri, requesterWebId)
         .then(connectionUris =>
             urisToLookupMap(
                 connectionUris,
@@ -1077,25 +1077,11 @@ const rdfstore = window.rdfstore;
     /*
      * Loads all URIs of a need's connections.
      */
-    won.getConnectionUrisOfNeed = (needUri) =>
-        won.getNode(needUri)
-            .then(need =>
-                won.getNode(need.hasConnections)
-            )
-            .then(connectionContainer => {
-                /*
-                 * if there's only a single rdfs:member in the event
-                 * container, getNode will not return an array, so we
-                 * need to make sure it's one from here on out.
-                 */
-                return is('String', connectionContainer.member) ?
-                    [connectionContainer.member] :
-                    connectionContainer.member
-            })
-            .catch(error => {
-                error.message = "Failed to fetch connection uris for need <" + needUri + "> because of:\n" + error.message;
-                throw(error);
-            });
+    won.getConnectionUrisOfNeed = (needUri, requesterWebId) =>
+        won.executeCrawlableQuery(won.queries["getAllConnectionUrisOfNeed"], needUri, requesterWebId)
+            .then(
+                (result) =>  result.map( x => x.connection.value));
+
 
     /**
      *

